src/vc.rs

```diff
@@ -730,6 +737,10 @@ impl LinkedDataDocument for Credential {
         let mut loader = StaticLoader;
         json_to_dataset(&json, more_contexts.as_ref(), false, None, &mut loader).await
     }
+
+    fn to_value(&self) -> Result<Value, Error> {
+        Ok(serde_json::to_value(&self)?)
```

Comment:
Could you make this a blanket implementation? `impl NewTrait for (dyn LinkedDataDocument + Serialize)` Or remove the default error.

But maybe I'm not thinking of everything, it's just it can be a bit tedious to debug if you forget to implement it (which is not something you do every day :sweat_smile:)

Reply:
I'm not sure how to do it with a blanket implementation. Serialize is not object-safe.
Default error removed in 06aa9c1ea4c379a6f488189a63237c9dbb855db0. It's a breaking change to `LinkedDataDocument`  but we aren't using `LinkedDataDocument` outside out the `ssi` crate.